NSS-Day-Cohort-61/team-1-GiffyGram
# Mrs direct messages

#### Changes Made

1. Modified file `MessageForm.js` to include a direct message form to appear when the pen icon is clicked. pressing save in the form currently saves the message to the database.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout mrs-direct-messages
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > when pen icon is clicked, new html is loaded. the html will be removed when pressing either the x or the cancel button. 
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
